installation-and-setup/managed-app-configuration.md

```diff
@@ -46,6 +49,11 @@ defaults write com.progress-telerik.fiddler DefaultProxyPort -integer 8899
 ```
 The above explicitly sets the Fiddler Everywhere default proxy port to port 8899.
 
+Example for applying the **DefaultNetworkCaptureList** key with multiple CIRD addresses. The value must be set as a string:
+```sh
+defaults write com.progress-telerik.fiddler DefaultNetworkCaptureList "0.0.0.0/0:443 0.0.0.0/0:8084"
```

Comment:
perhaps add some information that IPv6 addresses must be added separately from IPv4 ones if you want to capture both protocols:
```suggestion
defaults write com.progress-telerik.fiddler DefaultNetworkCaptureList "0.0.0.0/0:443 ::/0:443 0.0.0.0/0:8084 ::/0:8084 127.0.0.1/8:4200 ::1/128:4200"
```